File: src/main/kotlin/com/netflix/java/refactor/gradle/Styling.java
package com.netflix.java.refactor.gradle;

import org.gradle.internal.logging.text.StyledTextOutput;

class Styling {
	static StyledTextOutput.Style Bold = StyledTextOutput.Style.UserInput;
    static StyledTextOutput.Style Green = StyledTextOutput.Style.Identifier;
    static StyledTextOutput.Style Yellow = StyledTextOutput.Style.Description;
    static StyledTextOutput.Style Red = StyledTextOutput.Style.Failure;
}

File: src/main/kotlin/com/netflix/java/refactor/gradle/Styling.kt
package com.netflix.java.refactor.gradle

import org.gradle.logging.StyledTextOutput

object Styling {
	val Bold = StyledTextOutput.Style.UserInput
    val Green = StyledTextOutput.Style.Identifier
    val Yellow = StyledTextOutput.Style.Description
    val Red = StyledTextOutput.Style.Failure
}
